src/styles/styles.js

```diff
@@ -363,11 +364,14 @@ const styles = {
         textAlignVertical: 'center',
     },
 
-    textInputReversed: addOutlineWidth({
-        backgroundColor: themeColors.heading,
-        borderColor: themeColors.text,
-        color: themeColors.textReversed,
-    }, 0),
+    textInputReversed: addOutlineWidth(
+        {
+            backgroundColor: themeColors.heading,
+            borderColor: themeColors.text,
+            color: themeColors.textReversed,
+        },
+        0,
+    ),
```

Comment:
Please only make material changes thanks. If you have prettier or an autofixer enabled please disable it.

Reply:
Done